src/libs/actions/Report.js

```diff
@@ -616,6 +617,29 @@ function createOptimisticChatReport(participantList) {
     };
 }
 
+/*
+ * Creates an optimistic IOU report with a randomly generated reportID
+ */
+function buildOptimisticIOUReport(total, chatReportID, currency, locale) {
+    const formattedTotal = NumberFormatUtils.format(locale,
+        total, {
+            style: 'currency',
+            currency,
+        });
+    return {
+        cachedTotal: formattedTotal,
+        chatReportID,
+        currency,
+        hasOutstandingIOU: true,
+        managerEmail: '__FAKE__',
+        ownerEmail: '__FAKE__',
+        reportID: ReportUtils.generateReportID(),
+        state: 'SUBMITTED',
+        stateNum: 1,
```

Comment:
We have a [CONST](https://github.com/Expensify/App/blob/77e5843c6a928b135b993166c073841b21610781/src/CONST.js#L287) for this, but state `SUBMITTED = 2` 🤔

Reply:
Weird. submitted does look like state 1 to me [in Auth](https://github.com/Expensify/Auth/blob/cb9cc668f1557d565f308c45def21e51fc640bc1/auth/lib/Report.h#L42). Is App incorrect? 😕 

**Auth**
<img width="482" alt="Screenshot 2022-09-06 at 10 21 26" src="https://user-images.githubusercontent.com/10736861/188598297-3c0e9ab1-6304-43f4-bd89-a4a2694a4046.png">

**App**
<img width="277" alt="Screenshot 2022-09-06 at 10 21 17" src="https://user-images.githubusercontent.com/10736861/188598324-f8f6afac-4090-460e-b8df-c612a1f2607b.png">

Reply:
Need to look into this further.

Reply:
What's the verdict here?

Reply:
this seems odd indeed.

Reply:
> What's the verdict here?

More confusion... I'm sure that we should be using `stateNum: 1` based on how report data is currently returned from Auth and stored in App: 

<img width="913" alt="Screenshot 2022-09-08 at 10 38 40" src="https://user-images.githubusercontent.com/10736861/189090401-aa9bb927-8a65-4f3d-a163-ad5feecbd182.png">

I think I should use `CONST.STATUS.SUBMITTED`, as changing this is way out of scope.